### ISSUE TITLE
Document PyQt benign errors to avoid confusion

### ISSUE THREAD

**Issue by aawdls** [dls-controls/dev-c7#46]
When loading a PyQt window I get
```
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
```
Initial googling suggests some interaction with the nvidia drivers of the host.

I haven't yet found a package I can install to provide swrast_dri.so

More investigation, and comparison to another app, needed.

**Comment by aawdls** [comment]
For comaprison with dls_pmaccontrol, I get the error but the window is displayed.

So perhaps the error can be tolerated.

**Comment by gilesknap** [comment]
I've seen these errors too with dls-pmac-control but they seem to be benign for me. Are they stopping your app working?

**Comment by aawdls** [comment]
I thought so, but given that dls-pmac-control works, probably it is something else.

**Comment by aawdls** [comment]
Sure enough, I found the real cause of my problem. It is not related to these messages.

It would be nice to understand exactly what they mean, but I'm not going to do that now.

**Comment by gilesknap** [comment]
TODO: add some notes on this in the docs

**Comment by gilesknap** [comment]
fixed in 2.0.4
